### DISCUSSION TITLE
Colour Questions for the VR Headset Age

### DISCUSSION THREAD

**Discussion by habemus-papadum** [colour-science/colour#714] [Q&A] [answered]
Thank you so much for providing this amazing resource!  As you probably know, a technical understanding of colour is not so easy for everyone, and having concrete code certainly is useful for someone like me.  

I have two conceptual questions about colour that I've never found an appropriate forum in which to inquire.  I hope it's not too egregious for me to ask them here (with a little motivation described at the end.)

**How Do Humans Perceive Monochromatic Saturation?**
Imagine an experimental setup where the subject(me) is wearing a VR headset (e.g. like the oculus rift) but with "magic colourimetry pixels" that can emit a single frequency of light.  The device is initially set to a frequency and intensity (i.e. photons/sec entering my eyes)  such that I perceive that "I am seeing 'a red' that is moderately bright".  Now assume the experimenter decreases the intensity to "0" -- no photons at all entering my eyes.  I assume at that point I would state that "I am seeing pitch black" (though I hesitate to assume anything about colour perception...)   However it is unclear to me what would happen in the other direction.  I assume if the experimenter gradually increased the intensity, I would at some point no longer be able to perceive any changes.  I'll refer to that as "saturation" -- at this point, would I claim to see "a pure, bright white" or  "a bright red" (or something else) ?  Would my eyes be damaged at this point, or does that happen well past saturation?  

It has always seemed to me that even though several colour spaces have a "luminance" type dimension, they seem to have different opinions about exactly how humans perceive colour in the saturated luminance region.  When the L value of Lab is saturated, there can still be a red, but HSL seems to imply we perceive white at saturation.  

(I have  a second question more directly related to VR headsets, but I'll ask later)

Cheers, nehal

**Comment by thomasmansencal** [comment, marked as answer]
Hi @habemus-papadum,

> I would at some point no longer be able to perceive any changes.

Yes this is correct, respective [Photopsin](https://en.wikipedia.org/wiki/Photopsin) of the stimulated cone-type will bleach and you will not be able to see until they re-generate.

>  would I claim to see "a pure, bright white" or "a bright red" (or something else) ?

You would perceive the colour resulting from the light stimulus that excited your visual system.

**WARNING: Anybody reading what follows, please NEVER EVER LOOK AT A LASER, it would irremediably damage your eyes.**

Assuming a monochromatic red laser shining in your eyeball, you would perceive a red colour, your L-cones would photobleach and your retina would be destroyed by too much focused radiant energy, remember that the lens effect is to focus light onto the retina.

This paper has some thresholds for damage: http://www2.ensc.sfu.ca/~glennc/e894/e894l9k

HSL is not a proper colourspace when it comes to colour science, it has nothing perceptual.

Hope that helps!

Cheers,

Thomas

**Comment by habemus-papadum** [comment]
Thanks -- That is very helpful!  The notion of  photobleaching is a new concept and helps clarify the details involved.  I'll add my second question in another issue.  Cheers, nehal
